server/routerlicious/Dockerfile

```diff
@@ -22,8 +22,12 @@ ENV TINI_VERSION v0.18.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 RUN chmod +x /tini
 
+# There are some packages outside the routerlicious folder that we need to build, so
+WORKDIR /usr/src
+COPY --from=root tools/changelog-generator-wrapper/package*.json tools/changelog-generator-wrapper/
+
 # Copy over and build the server
-WORKDIR /usr/src/server
+WORKDIR /usr/src/server/routerlicious
```

Comment:
In several places we map the `server/routerlicious` folder from the host to `usr/src/server` in the container, so I'd suspect this change _might_ breaking things somewhere (first example that comes to mind is starting a server "environment" in docker locally with `docker compose` from `server/` and/or from `server/routerlicious`).

Reply:
Do you have examples of places where we make that assumption?

Reply:
The problem is that r11s has to depend on something outside of the folder, so the relative path to the dep needs to match in and outside of the docker container. I don't know how to make it work one way in the docker image and have a different dep path in the image.

Reply:
It's a bit hacky, but what if we copy the changelog-generator-wrapper folder to `/usr` instead of `/usr/src` while building the `base` target, so the file dep in r11s sees it in the same place? And add a good comment about why.

Some examples of mappings are [here](https://github.com/microsoft/FluidFramework/blob/423a60bce7142064c30178a9fd4a9b307a437573/server/routerlicious/docker-compose.dev.yml#L5) or [here](https://github.com/microsoft/FluidFramework/blob/423a60bce7142064c30178a9fd4a9b307a437573/server/docker-compose.dev.yml#L27), and assumptions that that is the case [here](https://github.com/microsoft/FluidFramework/blob/423a60bce7142064c30178a9fd4a9b307a437573/server/docker-compose.yml#L7).

Reply:
@scottn12 I tried something different in my PR. If it works then it should address Alejandro's concerns.

Reply:
@tylerbutler, I was able to get it to work locally, so I copied those changes in my latest commit